### DISCUSSION TITLE
[@types/ssh2] needs typings for generateKeyPair()

### DISCUSSION THREAD

**Discussion by GeekCornerGH** [DefinitelyTyped/DefinitelyTyped#68899] [Issues with a @types package] [answered]
I'm trying to use `generateKeyPair` on the `ssh2` package, which can be imported as follow:
```js
const { utils: { generateKeyPair} } = require('ssh2');
```
Sadly, typings for that function (and generateKeyPairSync) are missing, can you update them? Thanks.

[a comment by an automated account is not shown]

**Reply by GeekCornerGH** [reply]
Sure thing, bot

**Comment by tristan-f-r** [comment]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68905

**Reply by GeekCornerGH** [reply, marked as answer]
Thank you, afer updating, I can indeed confirm it works. Thanks for the quick PR, and have a nice day!
